src/gis/components/Map.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { MapContainer, TileLayer } from 'react-leaflet';
 
+import './Map.css';
```

Comment:
Suggestion:
Just a suggestion, for me is better to keep these imports always explicit and not relative
```suggestion
import 'gis/components/Map.css';
```